docs/source/refs.bib

```diff
@@ -0,0 +1,10 @@
+@article{zhang2017beyond,
```

Comment:
nice idea this should order a bit our refs

Reply:
Not working for now, I guess this dhould be a future PR